swagger.yaml

```diff
@@ -94,9 +89,7 @@ paths:
         "200":
           description: Array of directory entities
           schema:
-            type: array
-            items:
-              $ref: "#/definitions/ResultObject"
+            $ref: "#/definitions/ResultObjects"
```

Comment:
Can this be renamed as `DirectoryResults` or something more specific? Also, I don't think the `pretty` parameters is needed anymore, can you remove it as well?

Reply:
Done